samples/connect-wpf/WpfFormatterMixins.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Microsoft.DotNet.Interactive.CSharp;
+using Microsoft.DotNet.Interactive.Formatting;
+using System;
+using System.IO;
+using System.Windows;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+
+namespace WpfConnect
+{
+    public static class WpfFormatterMixins
+    {
+        public static CSharpKernel UseWpf(this CSharpKernel kernel)
+        {
+            UseSolidColorBrushFormatter();
+            UseFrameworkElementFormatter();
+            return kernel;
+        }
+
+        private static void UseSolidColorBrushFormatter()
+        {
+            Formatter<SolidColorBrush>.Register((brush, writer) =>
+            {
+                var color = brush.Color;
+                string stringValue = $"#{color.R:X2}{color.G:X2}{color.B:X2}{color.A:X2}";
```

Comment:
I think the "DSL style" for `PocketView` is more readable.

Reply:
Agreed, fixed